Resources/Locale/en-US/flavors/flavor-profiles.ftl

```diff
@@ -263,6 +263,42 @@ flavor-complex-themartinez = like violets and lemon vodka
 flavor-complex-cogchamp = like brass
 flavor-complex-white-gilgamesh = like lightly carbonated cream
 flavor-complex-antifreeze = warm
+flavor-complex-absinthe = like anise
+flavor-complex-blue-curacao = like orange flowers
+flavor-complex-acidspit = like French battery acid
+flavor-complex-allies-cocktail = like medicinal tincture
+flavor-complex-aloe = like morning dew
+flavor-complex-amasec = like space marine sweat
+flavor-complex-andalusia = like sour molasses
+flavor-complex-b52 = like an Irish pub
+flavor-complex-bahama-mama = like sunbathing in the Caribbean
+flavor-complex-barefoot = like a cassis milkshake
```

Comment:
Is "cassis" in reference to Cassis, France?

Reply:
in reference to the fruit cassis (ribes nigrum)

Reply:
Very well, makes sense and doesn't need capitalization.